runtime/include/jvmti.h.m4

```diff
@@ -45,16 +45,18 @@ JNIEXPORT jint JNICALL Agent_OnAttach(JavaVM* vm, char *options, void *reserved)
 #define JVMTI_VERSION_1_1 0x30010100
 #define JVMTI_VERSION_1_2 0x30010200
 #define JVMTI_VERSION_1 (JVMTI_VERSION_1_0)
-#define JVMTI_VERSION_9  0x30090000
+ifelse(eval(JAVA_SPEC_VERSION >= 11), 1, [#define JVMTI_VERSION_9  0x30090000
```

Comment:
9 inside the 11 ifdef seems odd.

Reply:
`JVMTI_VERSION_9` is supported by `LTS JDK 11` but not `JDK 8`, this define should not appear within `JDK 8 jvmti.h`, since `JDK 9/10` are out of service, its home becomes `JDK11` instead. I guess this is the state of new Java version scheme.

Reply:
The less odd way is `> 8`, but either way is fine with me.

Reply:
I do think it's better as it is now.

Reply:
ah, just changed to `eval(JAVA_SPEC_VERSION > 8)`, please confirm if you want previous `>=11` version.

Reply:
GAC can decide.

Reply:
I am taking `JAVA_SPEC_VERSION > 8` for now which is consistent with the other checks in this PR for `JVMTI_VERSION_9` cases.